File: BackEnd_ExpressJS/app/models/logError.js
const sequelize = require('../config/database');
const { DataTypes } = require('sequelize');
const User = require('./user');

const LogError = sequelize.define('tblLogError', {
        ideLogError: {
            type: DataTypes.BIGINT,
            primaryKey: true,
        },
        strAddress: {
            type: DataTypes.TEXT,
            allowNull: false,
        },
        strAppOrigin: {
            type: DataTypes.STRING,
            allowNull: false,
        },
        strParameters: {
            type: DataTypes.TEXT,
            allowNull: false,
        },
        strError: {
            type: DataTypes.TEXT,
            allowNull: false,
        },
        ideUser: {
            type: DataTypes.BIGINT,
            allowNull: true,
        },
        timeStamp: {
            type: DataTypes.DATE,
            allowNull: false,
        },
    }, {
        // Activar/Desactivar las marcas de tiempo
        timestamps: false,
        // Usar el nombre de la tabla tal como está en la base de datos
        freezeTableName: true,
    });

LogError.belongsTo(User, { foreignKey: 'ideUser' });

module.exports = LogError;

File: BackEnd_Django/api/myModels/logError.py
from django.db import models
from .user import User

class LogError(models.Model):
    ideLogError = models.BigAutoField(primary_key=True)
    strAddress = models.TextField()
    strAppOrigin = models.CharField(max_length=255)
    strParameters = models.TextField()
    strError = models.TextField()
    ideUser = models.ForeignKey(User, on_delete=models.SET_NULL, null=True, blank=True)
    timeStamp = models.DateTimeField(auto_now=True)
    
    class Meta:
        db_table = 'tblLogError'
    
    def __str__(self):
        return self.ideLogError
